python/tvm/relay/frontend/tflite.py

```diff
@@ -743,6 +744,24 @@ def convert_split(self, op):
 
         return out
 
+    def convert_transpose(self, op):
+        """transpose implementation."""
+        try:
+            from tflite.Operator import Operator
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+        assert len(input_tensors) == 2, "input tensors length should be 2"
+        input_tensor = input_tensors[0]
+        input_tensor_idx = input_tensor.tensor_idx
+
+        in_expr = self.get_expr(input_tensor_idx)
+        out = _op.transpose(data=in_expr)
```

Comment:
You omit the parameter of `axes`, which is `input_tensors[1]` of `transpose`.

Reply:
Hi @FrozenGene 
Thanks for reminding, I update the code for the parameter of axes and add new test case